dot/parachain/collator-protocol/message_test.go

```diff
@@ -448,129 +432,129 @@ func TestHandleCollationMessageAdvertiseCollation(t *testing.T) {
-						CollatingPeerState: CollatingPeerState{
-							ParaID: parachaintypes.ParaID(6),
-						},
-					},
-				},
-			},
-			expectedMessage: networkbridgemessages.ReportPeer{
-				PeerID: peerID,
-				ReputationChange: peerset.ReputationChange{
-					Value:  peerset.WrongParaValue,
-					Reason: peerset.WrongParaReason,
-				},
-			},
-			errString: ErrInvalidAssignment.Error(),
-		},
-		{
-			// NOTE: prospective parachain mode and prospective candidates were added in V2,
-			// In V1, prospective parachain mode is disabled by and prospective candidates is nil
-			// In V2, prospective parachain mode is enabled by and prospective candidates is not nil
-			description: "fail with protocol mismatch is prospective parachain mode in" +
-				" enable but with got a nil value for prospective candidate",
-			advertiseCollation: collatorprotocolmessages.AdvertiseCollation(testRelayParent),
-			perRelayParent: map[common.Hash]PerRelayParent{
-				testRelayParent: {
-					assignment: &testParaID,
-					prospectiveParachainMode: parachaintypes.ProspectiveParachainsMode{
-						IsEnabled: true,
-					},
-				},
-			},
-			peerData: map[peer.ID]PeerData{
-				peerID: {
-					view: View{},
-					state: PeerStateInfo{
-						PeerState: Collating,
-						CollatingPeerState: CollatingPeerState{
-							ParaID: testParaID,
-						},
-					},
-				},
-			},
-			errString: ErrProtocolMismatch.Error(),
-		},
-		{
-			description:        "fail if para reached a limit of seconded candidates for this relay parent",
-			advertiseCollation: collatorprotocolmessages.AdvertiseCollation(testRelayParent),
-			perRelayParent: map[common.Hash]PerRelayParent{
-				testRelayParent: {
-					assignment: &testParaID,
-					collations: Collations{
-						// For Collator Protocol v1, we can only second one candidate
-						// at a time, so seconded limit would be 1
-						secondedCount: 1,
-					},
-				},
-			},
-			peerData: map[peer.ID]PeerData{
-				peerID: {
-					view: View{},
-					state: PeerStateInfo{
-						PeerState: Collating,
-						CollatingPeerState: CollatingPeerState{
-							ParaID: testParaID,
-						},
-					},
-				},
-			},
-			expectedMessage: networkbridgemessages.ReportPeer{
-				PeerID: peerID,
-				ReputationChange: peerset.ReputationChange{
-					Value:  peerset.UnexpectedMessageValue,
-					Reason: peerset.UnexpectedMessageReason,
-				},
-			},
-			activeLeaves: map[common.Hash]parachaintypes.ProspectiveParachainsMode{},
-			errString:    ErrSecondedLimitReached.Error(),
-		},
+		// {
+		// 	description:        "fail with unknown peer if peer is not tracked in our list of active collators",
+		// 	advertiseCollation: collatorprotocolmessages.AdvertiseCollation(testRelayParent),
+		// 	perRelayParent: map[common.Hash]PerRelayParent{
+		// 		testRelayParent: {},
+		// 	},
+		// 	errString: ErrUnknownPeer.Error(),
+		// },
+		// {
+		// 	description: "fail with undeclared para if peer has not declared its para id" +
+		// 		" and report the peer",
```

Comment:
I'm wondering why these tests have been commented out?  Also, running `TestHandleCollationMessageAdvertiseCollation` seems to be blocked when I try running it.

Reply:
I see these tests are no longer commented out, but they still seem to be blocked for some reason, the test is timing out in CI after 45 minutes.

Reply:
fixed them